common/culture/innovations/wc_cultural_innovations.txt

```diff
@@ -27,9 +27,17 @@ innovation_massive_soldiers = {
 		is_giant_culture_trigger = yes
 	}
 	modifier = {
-		levy_size = -0.5
-		advantage = 20
-		army_maintenance_mult = 0.15
+		men_at_arms_limit = -2
+		men_at_arms_maintenance = 0.12
+		skirmishers_damage_mult = 0.1
+		skirmishers_toughness_mult = 0.08
```

Comment:
Don't get why damage and toughness differ.

Reply:
Toughness is a more valuable stat, would you prefer they be the same?

Also I will remove the maa limit like we talked about and replace it with more maintenance cost.

Reply:
> Toughness is a more valuable stat, would you prefer they be the same?

How it's more valuable? Yes.